src/redux/todosSlice.ts

```diff
@@ -67,5 +67,5 @@ const todosSlice = createSlice({
   },
 });
 
-export const { addTodo, removeTodo, toggleTodo } = todosSlice.actions;
+export const {addTodo, removeTodo, toggleTodo} = todosSlice.actions;
```

Comment:
Did something happen with formatting here?

Reply:
Yes, I changed a couple of code formatting rules